lib/rules/naming/non-state-var-no-leading-underscore.js

```diff
@@ -0,0 +1,32 @@
+const BaseChecker = require('./../base-checker')
+const naming = require('./../../common/identifier-naming')
+
+const ruleId = 'non-state-var-no-leading-underscore'
+const meta = {
+  type: 'naming',
+
+  docs: {
+    description: `State variable name not start with underscore.`,
+    category: 'Style Guide Rules'
+  },
+
+  isDefault: false,
+  recommended: true,
```

Comment:
I don't think this should be in the recommended ruleset.

Reply:
sure, I copy pasted from other rule and neglected this one.